packages/api/src/external/carequality/document/create-outbound-document-retrieval-req.ts

```diff
@@ -30,9 +30,6 @@ export function createOutboundDocumentRetrievalReqs({
   const now = dayjs().toISOString();
   const user = getSystemUserName(initiator.orgName);
 
-  const getDocRefsOfGateway = (gateway: OutboundDocumentQueryResp["gateway"]) =>
```

Comment:
removing logic that requires homeCommunity id of response to be the same ass homeCommunityId of gateway

Reply:
will turn on surescripts

Reply:
risk here: we start throwing more errors on DRs, and potentially lead to increased processing time due to timeouts